tests/unit/lms/views/dashboard/pagination_test.py

```diff
@@ -0,0 +1,87 @@
+import json
+
+import pytest
+from h_matchers import Any
+from sqlalchemy import select
+
+from lms.js_config_types import Pagination
+from lms.models import Course
+from lms.validation import ValidationError
+from lms.views.dashboard.pagination import PaginationParametersMixin, get_page
+from tests import factories
+
+
+class TestGetPage:
+    def test_when_no_next_page(self, pyramid_request, db_session):
+        pyramid_request.parsed_params = {"limit": 100}
+        courses = factories.Course.create_batch(5)
+        query = select(Course)
+        db_session.flush()
+
+        items, pagination = get_page(pyramid_request, query, (Course.id,))
+
+        assert items == courses
+        assert pagination == Pagination(next=None)
+
+    def test_when_empty(self, pyramid_request):
+        pyramid_request.parsed_params = {"limit": 100}
+        query = select(Course).where(False)
+
+        items, pagination = get_page(pyramid_request, query, (Course.id,))
+
+        assert items == []
+        assert pagination == Pagination(next=None)
+
+    def test_calculates_next(self, pyramid_request, db_session):
```

Comment:
```suggestion
    def test_it_calculates_next(self, pyramid_request, db_session):
```

For this test and the next one, "test_it" is an appropriate prefix for the test name.